### PR TITLE
HVG-189: App Navigation, Contract Detail, Coverage Tab

### PR DESCRIPTION
**Pull request by hedvigoscar**
Item from design: [insurance_details_view_coverage]

### Checklist

- [X] Functionality is covered by an integration test
- [X] Functionality is accessible in engineering mode

### CODE REVIEW

File: app/src/main/java/com/hedvig/app/feature/insurance/ui/contractcoverage/PerilsAdapter.kt [hunk 1]
```diff
@@ -7,9 +7,11 @@ import androidx.fragment.app.FragmentManager
...
 import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.RequestBuilder
+import com.google.android.material.card.MaterialCardView
 import com.hedvig.android.owldroid.fragment.PerilFragment
 import com.hedvig.app.BuildConfig
 import com.hedvig.app.R
+import com.hedvig.app.feature.insurance.ui.detail.coverage.PerilBottomSheet
 import com.hedvig.app.util.extensions.isDarkThemeActive
 import com.hedvig.app.util.extensions.view.setHapticClickListener
 import kotlinx.android.synthetic.main.peril_detail.view.*
```
Comment (MathiasOhrstrom) [inline app/src/main/java/com/hedvig/app/feature/insurance/ui/contractcoverage/PerilsAdapter.kt:17]: ViewBinding

File: app/src/main/res/layout/contract_detail_coverage_row.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```
Comment (MathiasOhrstrom) [inline app/src/main/res/layout/contract_detail_coverage_row.xml:3]: Can this be LinealLayout?

**Review by MathiasOhrstrom — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/hedvig/app/feature/insurance/ui/contractcoverage/PerilsAdapter.kt (hunk 1 above)
Comment (hedvigoscar) [reply app/src/main/java/com/hedvig/app/feature/insurance/ui/contractcoverage/PerilsAdapter.kt:17]: This file will be deleted upon feature completion, hence I will do no improving maintenance on it. 🙂

File: app/src/main/res/layout/contract_detail_coverage_row.xml (hunk 2 above)
Comment (hedvigoscar) [reply app/src/main/res/layout/contract_detail_coverage_row.xml:3]: This layout is actually more or less identical to the previous one where I decided to go with `ConstraintLayout` over `LinearLayout`, hence my decision on that one still applies 🙂

**Pull request merged**